eras/conway/impl/src/Cardano/Ledger/Conway/TxInfo.hs

```diff
@@ -762,3 +764,26 @@ class
 
 instance ConwayEraPlutusTxInfo 'PlutusV3 ConwayEra where
   toPlutusChangedParameters _ x = PV3.ChangedParameters (PV3.dataToBuiltinData (toPlutusData x))
+
+-- | Translate a validity interval to POSIX time for Conway era, always using an open upper bound (strictUpperBound).
+transValidityInterval ::
+  Inject (ConwayContextError era) a =>
+  Tx era ->
```

Comment:
This function does not require it to be a `Tx`. It is safer to convert it to a `proxy`
```suggestion
  proxy era ->
```